src/pages/developers/tutorials.tsx

```diff
@@ -445,12 +448,13 @@ const TutorialPage = ({
                   color="text"
                   fontWeight="semibold"
                   fontSize="2xl"
-                  mr={{ base: 0, md: 24 }}
+                  me={{ base: 0, md: 24 }}
                   _after={{
-                    ml: 0.5,
-                    mr: "0.3rem",
-                    display: tutorial.isExternal ? "inline" : "none",
+                    ms: 0.5,
+                    me: "0.3rem",
+                    display: tutorial.isExternal ? "inline-block" : "none",
                     content: `"↗"`,
+                    transform: flipForRtl,
```

Comment:
This can be a little tricky to test since at the moment we're not rendering any fallback tutorials for languages that don't contain any.

If desired, can test locally by changing the English `langDir` to `rtl` inside `i18n.config.json`, then load `http://localhost:3000/developers/tutorials`... Content will be in English, but can verify arrow direction (should point leftward for `rtl`):
<img width="988" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/7a76d973-2753-49ce-ad85-4ecd1ee1640a">